charts/node/templates/serviceMonitor.yaml

```diff
@@ -42,3 +42,48 @@ spec:
   {{- toYaml .| nindent 4 }}
   {{- end}}
 {{- end }}
+---
+{{- if and .Values.node.serviceMonitor.enabled .Values.node.collator.isParachain }}
+{{ $fullname :=  include "node.fullname" . }}
+{{ $serviceLabels :=  include "node.serviceLabels" .  }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ $fullname }}-parachain
```

Comment:
The service monitor shouldn't be labelled `*-parachain`, instead it's the relay-chain (the optionally monitored one) that should be labelled `*-relaychain`. If you are running a parachain, you care less about monitoring the relay-chain but more about monitoring your own chain.